velox/dwio/dwrf/writer/Writer.cpp

```diff
@@ -35,7 +36,99 @@ namespace facebook::velox::dwrf {
 
 void WriterOptions::processConfigs(
     const config::ConfigBase& connectorConfig,
-    const config::ConfigBase& session) {};
+    const config::ConfigBase& session) {
+  std::map<std::string, std::string> configs;
```

Comment:
Can you construct a local HiveConfig object with connectorConfig and use existing hive method for access? thanks!